### ISSUE TITLE
Management Features

### ISSUE THREAD

**Issue by WamWooWam** [WamWooWam/Unicord#14]
Unicord needs to be able to edit channel and server settings. None of this stuff is particularly difficult to do in itself (minus the fucken audit log), but getting the UI right and such is hard, so this might take some time. 

- [ ] Channel editor
    - [x] Text channel settings
    - [ ] Voice channel settings
- [ ] Server editor
    - [ ] Overview
    - [ ] Moderation
    - [ ] Audit log (kill me now)
    - [ ] Roles 
    - [ ] Emoji
    - [ ] Members
    - [ ] Invites 
    - [ ] Bans

**Comment by WamWooWam** [comment]
So this needs more work than I thought. Namely I need a clean way to open popup windows, currently unicord has about 4 different ways of opening popups, and they're all a bit naff. So, here's a few options, and I want some feedback. 

Do I:
- Open new windows
    - Has the added advantage of meaning you can move them around, position them, work on multiple things at once, etc.
    - Not the best experience on mobile devices
- Create one unified popup system
    - If so, what should it look like? Can you show me some examples?
    - Keep in mind, again, it has to be usable on mobile.

**Comment by Phoenix616** [comment]
> Open new windows

Yes, please. Not being able to move the popups to the side and interact/take a look at the rest of the server without opening a new web browser tab is one of the major flaws the Vanilla client imo.
